lib/collections.js

```diff
@@ -25,12 +29,22 @@ var normalizedWords = [];
 // then store those keys in the array uniqueWords
 
 // The words stored as keys of a dictionary
-var uniqueWordsAsKeys = {};
 
 // The keys pulled out into a list
-var uniqueWords = [];
 
 /* your code starts here */
+var uniqueWordsAsKeys = {};
+for (var i = 0; i < normalizedWords.length; i++) {
+ uniqueWordsAsKeys[normalizedWords[i]] = true;
+ }
+var uniqueWords = [];
+var i = 0;
+for (var key in uniqueWordsAsKeys) {
+ uniqueWords[i] = key;
+ i++;
```

Comment:
This will work, syntactically, but it would have been more idiomatic to use `.push()`.